Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -2500,6 +2501,11 @@ static bool getEventTypeFromWebEvent(const WebEvent& event, NSEventType& eventTy
     return SharedBuffer::create(pdfData);
 }
 
+ScrollbarGutter PDFPlugin::scrollbarGutterStyle() const
+{
+    return { };
+}
```

Comment:
I would have thought this could just use the parent implementation but this was necessary to avoid a build issue

```
Undefined symbols for architecture arm64:
  "__ZNK7WebCore14ScrollableArea20scrollbarGutterStyleEv", referenced from:
      __ZTVN6WebKit9PDFPluginE in UnifiedSource59-mm.o
ld: symbol(s) not found for architecture arm64
```

Reply:
That can be fixed by adding WEBCORE_EXPORT to the function declaration in the base class instead.

Reply:
Fixed that now